apps/src/sites/studio/pages/scripts/show.js

```diff
@@ -101,6 +102,9 @@ function initPage() {
   const mountPoint = document.createElement('div');
   $('.user-stats-block').prepend(mountPoint);
 
+  const completedLessonNumber = queryParams('completedLessonNumber');
+  updateQueryParam('completedLessonNumber', undefined);
```

Comment:
We render links to levels on the unit overview page with the same query params as the page has, we don't want to maintain this query param after leaving this page, which is why I'm removing it here.

Reply:
If we revisit this at some point, I think a comment in the code here would be helpful.  Also, is there a unit test to cover this behavior?  I can imagine accidentally breaking this.

Reply:
No, we generally aren't unit testing our `show.js` files. But I do have an idea for a UI test that I could add, I'll do that today

Reply:
Oh, darn it, I keep forgetting about that limitation.